README.md

```diff
@@ -1 +1,28 @@
-# Address-Library-Sample
\ No newline at end of file
+# Address-Library-Sample
+
+The AddrArrayLib Library provides utility functions to work with address[] types
+
+### Version
+    Truffle v5.0.10 (core: 5.0.10)
+    Solidity v0.5.0 (solc-js)
+    Node v8.11.3
+    Web3.js v1.0.0-beta.37
+    Ganache v2.1.0
+
+### Setup
+* Download Ganache https://github.com/trufflesuite/ganache/releases/v2.1.0

```

Comment:
Would be nice to provide an alternative with Ganache in Docker (a simple `docker run` command would suffice) for those (like me) who have trouble running the Ganache desktop version